core/benches/consumer.rs

```diff
@@ -155,15 +166,30 @@ fn bench_process_and_record_transactions(bencher: &mut Bencher, batch_size: usiz
 
 #[bench]
```

Comment:
this makes me wish there was a `test_case`-like crate for benches 😢 
I guess these will only live until the feature is activated everywhere